packages/extension-commerce-bm-seo/src/setup-app.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2025, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+// Third-Party
+import React from 'react'
+// import {RouteProps} from 'react-router-dom'
+import {RouteProps, BeforeRouteMatchParams} from '@salesforce/pwa-kit-extension-sdk/types'
+// Platform Imports
+import {
+    ApplicationExtension,
+    withApplicationExtensionStore
+} from '@salesforce/pwa-kit-extension-sdk/react'
+import {applyHOCs} from '@salesforce/pwa-kit-extension-sdk/react/utils'
+import {withOptionalCommerceSdkReactProvider} from './components/with-optional-commerce-sdk-react-provider'
+// Local Imports
+import {Config} from './types'
+import {SliceInitializer} from '@salesforce/pwa-kit-extension-sdk/react'
+
+// Overridable Imports
+// Using the `overridable` loader means that you are opting in to the override module resolution flow. As a result this module
+// will be resolved by first looking in the base projects `overrides` folder then the overrides folders of any extensions configured
+// after this one. Only if no module is found will the referenced module in this project be used.
+import seoHOC from 'overridable!./components/seo-hoc'
```

Comment:
Lets follow standards of using the prefix of `with` for HOC's. And axe the "HOC" post fix. We would want to change the export in that file too if it's names seoHOC.